perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -477,14 +493,18 @@ def _CliBasedTests(output_results, metadata, vm, iteration_count,
                                 'iterations.')
 
   # Report various percentiles.
+  metrics_prefix = ''
+  if FLAGS.cli_size != 'normal':
+    metrics_prefix = '%s ' % FLAGS.cli_size
```

Comment:
I thought about this, but that means if existing dashboards based on the metrics names would be broken unless we change those dashboards to also look for the new metadata field (cli_test_size). I'd prefer to add new metric names here so those existing dashboards continue to work.